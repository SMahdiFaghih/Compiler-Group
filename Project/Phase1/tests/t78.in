0123 00x123 0123x123 01230x123