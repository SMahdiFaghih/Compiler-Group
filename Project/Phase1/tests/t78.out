T_INTLITERAL 0123
T_INTLITERAL 0
T_INTLITERAL 0x123
T_INTLITERAL 0123
T_ID x
T_INTLITERAL 123
T_INTLITERAL 01230
T_ID x
T_INTLITERAL 123